chia/consensus/block_creation.py

```diff
@@ -5,6 +5,7 @@
 
 import blspy
 from blspy import G1Element, G2Element
+from chia_rs import compute_merkle_root
```

Comment:
might want to rename to `compute_merkle_set_root`. Merkle root implies merkle tree which is the main data structure most people use. But this is not a merkle tree